src/commands/optimize.ts

```diff
@@ -152,6 +163,7 @@ export default class Optimize extends Command {
     const canMoveAll = report.moveAllToComponents?.length;
     const canRemove = report.removeComponents?.length;
     const canReuse = report.reuseComponents?.length;
+    const canIgnoreSchema = this.disableOptimizations?.includes(DisableOptimizations.SCHEMA);
```

Comment:
I don't think we need this. we always show this option.

Reply:
So, we allow the user to ignore the schema halfway through even if they didn't choose the option initially and vice versa?

Reply:
Yep, that's the idea.

Reply:
I included these changes in the last commit.